Makefile

```diff
@@ -122,3 +146,22 @@ doc: build-tools ## Generate the documentation
 	--build-arg=MAIN_BRANCH=$(MAIN_BRANCH) \
 	doc
 	MAJOR_VERSION=$(MAJOR_VERSION) MAIN_BRANCH=$(MAIN_BRANCH) ci/extract-documentation artifacts/documentations/
+
+
+.PHONY: transifex
+transifex:
+	python3 -m pip install --user transifex-client
+
+admin/c2cgeoportal_admin/locale/%/LC_MESSAGES/c2cgeoportal_admin.mo: transifex
+	rm /tmp/c2ctemplate-cache.json
+	make --makefile=build.mk $@
+
+ADMIN_PO_FILES = admin/c2cgeoportal_admin/locale/fr/LC_MESSAGES/c2cgeoportal_admin.mo
+.PHONY: dev
+dev: $(ADMIN_PO_FILES) ## Generate a development environment that can be mount in a project container
+	(cd admin; npm install)
```

Comment:
Not sure to understand why we need this.
Admin JS packages might be installed in /opt in Docker images.

Reply:
Effectively, removed :-)